pkg/controllermanager/controller/shoot/shoot_control_reconcile.go

```diff
@@ -96,10 +96,15 @@ func (c *defaultControl) reconcileShoot(o *operation.Operation, operationType ga
 			Fn:           flow.SimpleTaskFn(botanist.DeployCloudProviderSecret).RetryUntilTimeout(defaultInterval, defaultTimeout),
 			Dependencies: flow.NewTaskIDs(deployNamespace),
 		})
+		createEtcdEncryptionConfiguration = g.Add(flow.Task{
+			Name:         "Creating EncryptionConfiguration for etcd encryption",
+			Fn:           flow.SimpleTaskFn(botanist.CreateEtcdEncryptionConfiguration),
```

Comment:
```suggestion
			Fn:           flow.TaskFn(botanist.CreateEtcdEncryptionConfiguration),
```

`SimpleTaskFn` is deprecated- for new functions we use `TaskFn`